src/SonarQube.MSBuild.Tasks/SonarQube.MSBuild.Tasks.csproj

```diff
@@ -41,22 +37,15 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\..\packages\SonarAnalyzer.CSharp.6.6.0.3969\analyzers\SonarAnalyzer.CSharp.dll" />
-    <Analyzer Include="..\..\packages\SonarAnalyzer.CSharp.6.6.0.3969\analyzers\SonarAnalyzer.dll" />
-    <Analyzer Include="..\..\packages\SonarAnalyzer.CSharp.6.6.0.3969\analyzers\Google.Protobuf.dll" />
-  </ItemGroup>
-  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="14.3.0" />
```

Comment:
Why is this ref now required? The version of MSBuild referenced in lines 42 and 43 below has not changed.

Reply:
It's not - now removed. Thanks!